operator/src/main/resources/scripts/commonMount.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+# Init container script for the common mount feature. 
+# See 'domain.spec.serverPod.commonMount' for details.
+
+scriptDir="$( cd "$(dirname "$0")" > /dev/null 2>&1 ; pwd -P )"
+
+if [ "${debug}" == "true" ]; then set -x; fi;
+
+source ${scriptDir}/utils.sh
+[ $? -ne 0 ] && echo "[SEVERE] Missing file ${scriptDir}/utils.sh" && exit 1
+
+if [ -z ${COMMON_MOUNT_COMMAND} ]; then
+  trace ERROR "Common Mount: The 'serverPod.commonMount.container.mountCommand' is empty for the container image='$CONTAINER_IMAGE'. Exiting"
+  exit
+fi
+
+trace INFO "Common Mount: About to execute command '$COMMON_MOUNT_COMMAND' in container image='$CONTAINER_IMAGE'. COMMON_MOUNT_PATH is '$COMMON_MOUNT_PATH' and COMMON_TARGET_PATH is '${COMMON_TARGET_PATH}'."
```

Comment:
We should probably avoid INFO. It's too verbose because INFO and above are automatically copied into the operator log by default. Please use FINE instead, plus modify the trace ERROR at line 29 to also mention the COMMON_MOUNT_PATH.

I should have thought of this earlier (sorry).

Reply:
Done.